content/foundation/working-groups/index.md

```diff
@@ -0,0 +1,10 @@
++++
```

Comment:
This is great, thanks. We'll want to document the WG process, but I can do that in a second PR, lets not block this.